### ISSUE TITLE
Incorrect classification example usage for Linux (azurerm_automation_software_update_configuration)

### ISSUE THREAD

**Issue by chris-fison** [hashicorp/terraform-provider-azurerm#19430]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Latest

### AzureRM Provider Version

Latest

### Affected Resource(s)/Data Source(s)

azurerm_automation_software_update_configuration

### Terraform Configuration Files

```hcl
provider "azurerm" {
  features {}
}

resource "azurerm_resource_group" "test" {
  name     = "example-rg"
  location = "East US"
}

resource "azurerm_automation_account" "test" {
  name                = "example"
  location            = azurerm_resource_group.test.location
  resource_group_name = azurerm_resource_group.test.name
  sku_name            = "Basic"
}

resource "azurerm_automation_software_update_configuration" "example" {
  name                  = "example"
  automation_account_id = azurerm_automation_account.test.id
  operating_system      = "Linux"

  linux {
    classification    = "Security"
    excluded_packages = ["apt"]
    included_packages = ["vim"]
    reboot            = "IfRequired"
  }

  duration = "PT2H2M2S"
}
```

### Debug Output/Panic Output

```shell
resource "azurerm_automation_software_update_configuration" "example" {
  name                  = "example"
  automation_account_id = azurerm_automation_account.test.id
  operating_system      = "Linux"

  linux {
    classification    = "Security"
    excluded_packages = ["apt"]
    included_packages = ["vim"]
    reboot            = "IfRequired"
  }

  duration = "PT2H2M2S"
}
```

### Expected Behaviour

Running above code should produce a Linux configuration for AUM.

### Actual Behaviour

Plan fails as linux block classification needs changing to below for the example else plan returns error saying classification type is invalid. So change exmaple from;

 linux {
    classification    = "Security"
    excluded_packages = ["apt"]
    included_packages = ["vim"]
    reboot            = "IfRequired"
  }

to

 linux {
    classification_included   = "Security"
    excluded_packages = ["apt"]
    included_packages = ["vim"]
    reboot            = "IfRequired"
  }

### Steps to Reproduce

Copy/paste following code obtained from (https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/automation_software_update_configuration) and run in Terraform, itll error saying classification not expected here.

provider "azurerm" {
  features {}
}

resource "azurerm_resource_group" "test" {
  name     = "example-rg"
  location = "East US"
}

resource "azurerm_automation_account" "test" {
  name                = "example"
  location            = azurerm_resource_group.test.location
  resource_group_name = azurerm_resource_group.test.name
  sku_name            = "Basic"
}

resource "azurerm_automation_software_update_configuration" "example" {
  name                  = "example"
  automation_account_id = azurerm_automation_account.test.id
  operating_system      = "Linux"

  linux {
    classification    = "Security"
    excluded_packages = ["apt"]
    included_packages = ["vim"]
    reboot            = "IfRequired"
  }

  duration = "PT2H2M2S"
}

### Important Factoids

No

### References

Example is shown in following link; https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/automation_software_update_configuration

**Comment by xyq175com** [comment]
I can works on this and drop a PR later

**Comment by hypery2k** [comment]
@xyq175com would be nice to get the docs updated ;)

**Comment by xyq175com** [comment]
> @xyq175com would be nice to get the docs updated ;) 

Sure will drop PR later.

**Comment by stephybun** [comment]
Closed by #20624
